go-controller/README.md

```diff
@@ -30,7 +30,7 @@ Then find the executables here : go-controller/_output/go/windows/
 ### Usage
 
 Run the 'ovnkube' executable to initialize master, node(s) and as the central all-in-one controller that builds the network as pods/services/ingress objects are born in kubernetes.
-Options specified on the command-line override both defaults and configuration file options.
+Options specified on the command-line override environment variables. Environment variables override service account files and defaults and configuration file options.
```

Comment:
Env variables don't override config file options, according to the code. The `overrideFields(&Kubernetes, &file.Kubernetes)` function will use config file info to override anyhting that came before, including SA, env, and defaults.

Reply:
Changed.